### ISSUE TITLE
Values migration may leave incorrect data

### ISSUE THREAD

**Issue by j-zimnowoda** [redkubes/otomi-core#757]
**Describe the bug:** (a clear and concise description of what the bug is)

**To Reproduce**
Having files:
teams.yaml
```
teamConfig:
    teams: {}
```
and 
secrets.teams.yaml
```
teamConfig:
    teams: {}
```

and relocation 
```
    relocations:
      - teamConfig.teams: teamConfig
```
After running `otomi migrate`
The `secrets.teams.yaml` will preserve old structure.
The `teams.yaml` is emptied

**Expected behavior:** (a clear and concise description of what you expected to happen)
Both files are emptied.

**Screenshots:** (if applicable, add screenshots to help explain your problem)

**Cluster(s):**

- Cloud: [e.g. aws]
- Cluster: [e.g. demo]

**Versions:** (seen in console's `/settings`)

- Core Version [e.g. 0.12.34]
- Console Version: [e.g. aws]
- API version: [e.g. 0.4.56]

**Desktop:** (if relevant)

**Smartphone:** (if relevant)

**Additional context:** (add any other context about the problem here)

**Comment by j-zimnowoda** [comment]
@Morriz perhaps you know how to fix it

**Comment by j-zimnowoda** [comment]
@Morriz perhaps you know how to fix it

**Comment by Morriz** [comment]
If there is no teams data nothing is done with that secrets file when `writeValues` is called.

**Comment by j-zimnowoda** [comment]
ok, no worries if it is not an easy fix.
This is highly unlikely to happen because who is going to upgrade otomi without any team deployed ? 
I am closing it because there are more important things to tackle.

**Comment by Morriz** [comment]
Fixed in this PR: #761
